recipes/dash-html-components/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "dash-html-components" %}
+{% set filename = "dash_html_components" %}
+{% set version = "0.10.1" %}
+{% set sha256 = "b6460e0eba55da6abeb6e52b0b4fca066db3e9fdc8866d44f6b71a2b74c978b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ filename }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - dash_html_components
+    - dash_html_components.version
+
+about:
+  home: https://plot.ly/dash
+  dev_url: https://github.com/plotly/dash-html-components
+  license: MIT
+  license_family: MIT
```

Comment:
Can you manually package the `license_file` and add the following link as a comment so that if/when it gets merged, there is a visual reminder to update the feedstock later:
https://github.com/plotly/dash-html-components/pull/46

https://conda-forge.org/docs/meta.html#packaging-the-license-manually